### ISSUE TITLE
 UUID.randomUUID().toString() is not detected

### ISSUE THREAD

**Issue by maschnetwork** [aws/aws-lambda-snapstart-java-rules#16]
Hi,

it seems like that UUID.randomUUID().toString() is not detected. See below example.

**WORKS** (detects an issue):

```java
@Component
public class LambdaHandler implements RequestHandler<String, String> {

    private final UUID random = UUID.randomUUID();

    @Override
    public String handleRequest(String event, Context context) {
        return random.toString();
    }
}
```

**DOES NOT WORK** (detects no issues):

```java
@Component
public class LambdaHandler implements RequestHandler<String, String> {

    private final String random = UUID.randomUUID().toString();

    @Override
    public String handleRequest(String event, Context context) {
        return random;
    }
}
````
